app/vlinsert: loki healthcheck endpoint (#7864)
VictoriaMetrics/VictoriaMetrics: docs/VictoriaLogs/CHANGELOG.md

* FEATURE: [`facets` pipe](https://docs.victoriametrics.com/victorialogs/logsql/#facets-pipe): add an ability to return log fields with the same values across all the selected logs by adding `keep_const_fields` option. Such log fields aren't interesting in most cases, so they aren't returned by default.
* FEATURE: [HTTP querying APIs](https://docs.victoriametrics.com/victorialogs/querying/#http-api): allow passing arbitrary [LogsQL filters](https://docs.victoriametrics.com/victorialogs/logsql/#filters) to `extra_filters` and `extra_stream_filters` query args. See [these docs](https://docs.victoriametrics.com/victorialogs/querying/#extra-filters) and [this feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5542) for details.

> * FEATURE: [Grafana Loki data ingestion](https://docs.victoriametrics.com/victorialogs/data-ingestion/promtail/): added `/insert/ready` endpoint for Loki healthcheck. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7824).

Reviewer: ```suggestion
* FEATURE: [Grafana Loki data ingestion](https://docs.victoriametrics.com/victorialogs/data-ingestion/promtail/): add support of Loki healthcheck `/insert/ready` endpoint. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7824).
```

Please, prefer using present tense for verbs in commit messages and changelog. Use the imperative, present tense: "change" not "changed" nor "changes". Read your commit message as "This commit will .."
don't capitalize the first letter.

Reviewer: Added this here https://github.com/VictoriaMetrics/VictoriaMetrics/commit/bbc84fa11926561f8da24f58cc3fc2d98c872edb

> 

Reviewer: ```suggestion
```
